tests/__init__.py

```diff
@@ -46,4 +46,4 @@ def teardown_package():
     cur = conn.query('SHOW DATABASES LIKE "{}\_%%"'.format(PREFIX))
     for db in cur.fetchall():
         conn.query('DROP DATABASE `{}`'.format(db[0]))
-    conn.query('SET FOREIGN_KEY_CHECKS=1')
+    conn.query('SET FOREIGN_KEY_CHECKS=1').close()
```

Comment:
Are you sure that addition of `.close()` is a safe operation? The connection object is largely shared throughout the test so we have to ensure that it doesn't get closed in the middle.

Reply:
This does not close the connection but the cursor.  However, I experimented with closing the cursor after every query to attempt to address #515 but I do not think that was the problem. I took all `.close()` out but forgot this one.

Reply:
Each query creates its own cursor so closing the cursor should not make any difference

Reply:
Fair